protocol/flatbuffers/rpc.fbs

```diff
@@ -114,7 +114,7 @@ enum SkeletonBone: uint8 {
     LEGS_LENGTH = 8,
     KNEE_HEIGHT = 9,
     FOOT_LENGTH = 10,
-    FOOT_OFFSET = 11,
+    FOOT_SHIFT = 11,
```

Comment:
IMO shift is not clear, offset is.

Reply:
FOOT_SHIFT shifts the foot's head node backwards or forward, affecting the whole foot and not just its computed tracker.

OFFSETs offset tracker nodes only, without affecting the rest of the skeleton.